### PR TITLE
T#543/scoreboard

### PR DESCRIPTION
**Pull request by sudo-may**

### CODE REVIEW

File: client-display/src/userSkills/pointProgress/PointProgressChart.vue [hunk 1]
```diff
@@ -29,15 +29,17 @@ limitations under the License.
...
     <div class="card-body m-0 mr-1 p-0 apex-chart-container">
       <apexchart ref="ptChart" id="points-chart" v-if="!loading.inProgress && hasData" :options="chartOptions"
                  :series="chartSeries" height="200" type="area" />
       <span v-if="animationEnded" data-cy="pointHistoryChart-animationEnded"></span>
       <point-history-chart-placeholder v-if="loading.inProgress || !hasData" />
     </div>
-    <button v-if="chartWasZoomed" @click="resetZoom"
-            class="reset-zoom-btn btn btn-outline-primary btn-sm"><i class="fas fa-search-minus"></i> Reset Zoom</button>
+<!--    <button v-if="chartWasZoomed" @click="resetZoom"-->
+<!--            class="reset-zoom-btn btn btn-secondary btn-sm skills-theme-btn"><i class="fas fa-search-minus"></i> Reset Zoom</button>-->
```
Comment (rmmayo) [inline client-display/src/userSkills/pointProgress/PointProgressChart.vue:42]: should these lines be removed?

File: service/src/test/java/skills/intTests/metrics/skill/UsagePostAchievementMetricsBuilderSpec.groovy [hunk 2]
```diff
@@ -24,6 +24,7 @@ import skills.metrics.builders.skill.UsagePostAchievementMetricsBuilder
 import skills.services.LockingService
 import skills.services.UserEventService
 import skills.storage.repos.UserAchievedLevelRepo
+import spock.lang.IgnoreRest
```
Comment (rmmayo) [inline service/src/test/java/skills/intTests/metrics/skill/UsagePostAchievementMetricsBuilderSpec.groovy:27]: un-used import

**Review by rmmayo — CHANGES_REQUESTED** [review]

File: client-display/src/userSkills/pointProgress/PointProgressChart.vue (hunk 1 above)
Comment (sudo-may) [reply client-display/src/userSkills/pointProgress/PointProgressChart.vue:42]: done

File: service/src/test/java/skills/intTests/metrics/skill/UsagePostAchievementMetricsBuilderSpec.groovy (hunk 2 above)
Comment (sudo-may) [reply service/src/test/java/skills/intTests/metrics/skill/UsagePostAchievementMetricsBuilderSpec.groovy:27]: done

**Review by rmmayo — APPROVED** [review]

**Pull request merged**